Meeting-Material/readme.md

```diff
@@ -0,0 +1,33 @@
+# Guide for meeting chairs
+This guide is meant to help to keep the effort as low as possible for the meeting chair while keeping the documentation as accurate as possible at the same time.
+
+## Before the Meeting 
+1. create a preparation branch "meeting_yyyymmdd" with yyyymmdd the date of the next meeting
+2. in the branch create a new file in a respective subfolder /Meeting-yyyymmdd/yyyymmdd.md
+3. you can use the following file [tbd] as template for the content of the new .md meeting log
+4. Check the mailing list for topics relevant for the meeting - potentially request topics by sending a mail to the list (may also serve as reminder)
+5. Prepare the standard content including the agenda
+6. Prepare a web-conference system (ideally BigBlueButton https://conf.fsfe.org/b/compliance-tooling see hints below)
+7. (optional) Check the OpenChain Global Calendar if the meeting details are correctly presented there and if necessary contact him for update
+8. Create a meeting invitation including the agenda and send it to the mailing list 
+
+## While the Meeting 
+1. Start the meeting in the respective meeting tool (ideally Big Blue Button)
+2. (optional) Ask / inform all participants for consent for recording and turn the cameras off. Start recording
+3. Show the Antitrust slide from OpenChain and inform about the Antitrust requirements (e.g. "Stick to the shared agenda", ...). 
+4. Take notes in the yyyymmdd.md in the branch set up above in "2."
+5. Collect all participant names in the notes.
```

Comment:
Why is this needed?

Reply:
When I made the first *.md-file I tried to copy structure from a good practice so I went to the SPDX-community: https://github.com/spdx/meetings/tree/main/general
They have this section but tbh I had the same question but didn't have the time yet to ask them nor did I have the time to check if our internal antitrust requirements would require that transparency (If we manage to come back to record the meetings, the whole meeting would be anyway transparent).
I tried it out in my sessions to list the names and with MS Teams it was rather easy to collect the names but it is additional effort to invest. So I appreciate that you brought up this point. Also the agenda-point of "approving" the minutes of the last meeting seems to me very formal, but on the other hand the SPDX-meeting documentation is really crystal clear :-)

Reply:
@mkurzman thank you for the explanation. I would not regard this as mandatory. Approving the minutes I would not introduce. We are an open group and persons who participated in meeting a might not be present in meeting a+1 who will be empowered to approve the minutes in this case. I think this is overhead with zero value, so lets skip it.